.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
```

Comment:
this is number of days of inactivity before a PR becomes stale?

Reply:
Yes. Activity is defined as:

> Any change to an issues and pull request is considered an update, including comments, changing labels, applying or removing milestones, or pushing commits.